conda_smithy/configure_feedstock.py

```diff
@@ -2601,7 +2594,7 @@ def get_migrations_in_dir(migrations_root):
     from the timestamp to a tuple of (filename, migration_number)
     """
     res = {}
-    for fn in glob.glob(os.path.join(migrations_root, "*.yaml")):
+    for fn in list(Path(migrations_root).glob("*.yaml")):
```

Comment:
I don't think the `list()` call is needed here; `glob()` returns a generator that can be consumed directly by a `for` loop.